builtin/logical/pki/storage.go

```diff
@@ -265,11 +279,11 @@ func importKey(ctx context.Context, b *backend, s logical.Storage, keyValue stri
 
 	foundExistingKeyWithName := false
 	for _, identifier := range knownKeys {
-		existingKey, err := fetchKeyById(ctx, s, identifier)
+		existingKey, err := sc.fetchKeyById(identifier)
 		if err != nil {
 			return nil, false, err
 		}
-		areEqual, err := comparePublicKey(ctx, b, existingKey, pkForImportingKey)
+		areEqual, err := comparePublicKey(sc.Context, sc.Backend, existingKey, pkForImportingKey)
```

Comment:
Maybe `comparePublicKey()`  could be a `storageContext` method as well?

Reply:
I updated it to take a storageContext parameter, but I followed @stevendpclark's guidance and didn't make this a `sc` member.